# file: C:\Users\ADITI\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\scipy\special\_multiufuncs.py
# hypothesis_version: 6.135.14

['->', 'assoc_legendre_p', 'assoc_legendre_p_all', 'axes', 'diff_n', 'dtype', 'legendre_p', 'legendre_p_all', 'n', 'out', 'resolve_dtypes', 'resolve_out_shapes', 'sph_harm_y', 'sph_harm_y_all', 'sph_legendre_p', 'sph_legendre_p_all']